### DISCUSSION TITLE
Issue images aren’t showing

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69518] [Mobile]
Issue example: https://github.com/AdguardTeam/AdGuardDNS/issues/675

**What I should be seeing**
![image](https://github.com/community/community/assets/95468948/7695c3c5-29b0-4238-8d67-4b4ff2fb37d4)

**What I'm actually seeing**
![image](https://github.com/community/community/assets/95468948/593d2afe-7630-47db-b2e6-216fa151cc72)

<details/>

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone 13 Pro (iOS 17.0.3)
</details>

**Comment by Iron-Ham** [comment]
Hello @jslawler,

Are you still running into this with that particular issue? If so, are there any particular reproduction steps I should follow? I am unable to reproduce on my end (see video below).

https://github.com/community/community/assets/3388381/1a0022c4-db89-4db4-ba7e-5494a57c6491

Thanks,
Hesham
